pkgs/tools/graphics/amber/default.nix

```
@@ -0,0 +1,101 @@
+{ stdenv
+, fetchgit
+, fetchFromGitHub
+, cmake
+, git
+, pkgconfig
```

```suggestion
, pkg-config
```

`pkgconfig` is an alias, `pkg-config` should be used.